images/virtualization-artifact/pkg/controller/indexer/vi_indexer.go

```diff
@@ -58,3 +58,18 @@ func IndexVIByStorageClass() (obj client.Object, field string, extractValue clie
 		}
 	}
 }
+
+func IndexVIByPhaseAndStorage() (obj client.Object, field string, extractValue client.IndexerFunc) {
```

Comment:
Same as for CVI: https://github.com/deckhouse/virtualization/pull/1622/changes#r2646879374

The name should be more specific.